client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -27,11 +27,86 @@
         style="max-height: 52vh;"
       />
     </b-form-group>
+
+    <div>
```

Comment:
As mentioned style this to have a bigger space, consider adding an "\<hr\>" element between them